### PR TITLE
Bdd delete

### PR DESCRIPTION
**Pull request by nideyidayida**

### CODE REVIEW

File: features/inventory.feature [hunk 1]
```diff
@@ -15,10 +15,16 @@ Scenario: Retrive an ProductInformation
     And I set the "prod_id" to "2"
     And I press the "retrieve" Button
     Then I should see "2" in the "new_qty" field
-    And I should see "2" in the "used_qty" field
-    And I should see "2" in the "open_boxed_qty" field
-    And I should see "5" in the "restock_level" field
-    And I should see "9" in the "restock_amt" field
+    AND I should see "2" in the "used_qty" field
```
Comment (MingshiPeng) [inline features/inventory.feature]: Please merge with the latest master branch first.

Comment (jockyxu1122) [reply features/inventory.feature]: done

Comment (MingshiPeng) [reply features/inventory.feature]: It seems the 'AND' still exists in branch bdd_delete, could you change them to 'And' instead?

Comment (jockyxu1122) [reply features/inventory.feature]: Addressed.

**Review by MingshiPeng — APPROVED** [review]

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180429-025222 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180429-025222") to [dev](https://console.bluemix.net/apps/1db994d8-c4f1-4076-85c0-7ff2674244e5?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:dev"), including commit ec9dcde019857f865ce787d92508c42c8d334a49

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-183630 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-183630") to [prod](https://console.bluemix.net/apps/e25e653b-a50f-4831-b93e-d8f9f484a484?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:prod"), including commit ec9dcde019857f865ce787d92508c42c8d334a49

**Pull request merged**